### ISSUE TITLE
Think about variable annotations

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#457]
# Rule request

## Thesis

I have several concerns about this code:

```python
def function():
    some_var: int
    ...
```

1. Variable `some_var` is not initialized properly
2. It does not have any effect on runtime
3. It shows that there are some type related problems with this function

Should we forbid this kind of expressions in our code?
What are valid use-cases for this?

Valid usages:
1. inside classes, as class attributes

**Comment by sobolevn** [comment]
We should not mess with types.
